src/App.js

```diff
@@ -5,15 +5,21 @@ import './App.css';
 import Header from './components/Header/Header';
 import Footer from './components/Footer/Footer';
 import Main from './components/Main/Main';
+//Importing background to set background image// 
+import background from './background.png';
+
 
 function App() {
   return (
-    <div className="App">
-      <Header />
-      <Main />
-      <Footer />
+    <div className="App" style={{ backgroundImage: `url(${background})` }}>
+      <div className='animal'>
+        <Header />
+        <Main>
+
+        </Main>
```

Comment:
In our main.js file is where we will import animals and utilize .map to map over each animal and return the Animal component. Where the Animal component is called in Main.js, we can pass down our props to the Animal component within Animal.js. Please let me know if you need any help with this, it can be quite confusing in the beginning.